contrib/kind.sh

```diff
@@ -42,6 +42,7 @@ usage() {
     echo "                 [-dbl|--dbchecker-loglevel <num>] [-ndl|--ovn-loglevel-northd <loglevel>]"
     echo "                 [-nbl|--ovn-loglevel-nb <loglevel>] [-sbl|--ovn-loglevel-sb <loglevel>]"
     echo "                 [-cl |--ovn-loglevel-controller <loglevel>] [-dl|--ovn-loglevel-nbctld <loglevel>] |"
+    echo "                 [-tc |--ovn-log-to-tcpdump]"
```

Comment:
I'd call it -td and --ovndb-etcd-tcpdump

Reply:
Done.Thanks